hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/states/datanode/InitDatanodeState.java

```diff
@@ -178,4 +178,9 @@ public DatanodeStateMachine.DatanodeStates await(long time,
       ExecutionException, TimeoutException {
     return result.get(time, timeUnit);
   }
+
+  @Override
+  public void clear() {
+
+  }
```

Comment:
No change needed in this file after adding default no-op implementation.

Reply:
I have updated it.